clients/pkg/promtail/targets/kafka/target_syncer.go

```diff
@@ -262,17 +306,32 @@ func (ts *TargetSyncer) Stop() error {
 	return ts.close()
 }
 
+// ActiveTargets returns active targets from its consumer
+func (ts *TargetSyncer) ActiveTargets() []target.Target {
+	return ts.getActiveTargets()
+}
+
+// DroppedTargets returns dropped targets from its consumer
+func (ts *TargetSyncer) DroppedTargets() []target.Target {
+	return ts.getDroppedTargets()
+}
```

Comment:
Now that these exists, should we update the `kafka/targetmanager.go` to use these instead of the embedded consumer ones?

Reply:
Nice catch 👍